pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -151,20 +131,51 @@ func (m *SKRWebhookManifestManager) Install(ctx context.Context, kyma *v1beta2.K
 	return nil
 }
 
+func (m *SKRWebhookManifestManager) verifyCertNotRenew(certificate *certmanagerv1.Certificate,
```

Comment:
Just some weird name, it actually just updates watcher metrics and does not verify anthing since it also doesnt' return anything. Can you rename it to tell more what it does?

Reply:
changed to `updateCertNotRenewMetrics`